src/slave/containerizer/mesos/isolators/network/cni/cni.cpp

```diff
@@ -428,6 +428,10 @@ Future<Nothing> NetworkCniIsolatorProcess::recover(
       }
     }
 
+    if (infos.contains(containerId)) {
```

Comment:
This will make the unknown orphaned containers are skipped (i.e., we will miss to do cleanup for unknown orphaned container). So I would suggest to just do a `continue;` between L420 and L421.

@vixns Do you want to post a patch in https://reviews.apache.org/?